cli/bench/config.go

```diff
@@ -0,0 +1,48 @@
+package bench
+
+import (
+	"fmt"
+
+	"github.com/FZambia/tarantool"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+var bench_space_name = "__benchmark_space__"
+var bench_space_primary_index_name = "__bench_primary_key__"
+
+func print_operation_count(operations_map []Operaiton) {
+	for _, operation := range operations_map {
+		fmt.Println("	"+operation.name+" count: ", operation.count)
+	}
+}
+
+func print_operation_percent(operations_map []Operaiton) {
+	for _, operation := range operations_map {
+		fmt.Printf("\t%s: %2.0f%%\n", operation.name, operation.percent*100)
+	}
+}
+
+func print_operation_connections_count(operations_map []Operaiton) {
+	for _, operation := range operations_map {
+		fmt.Println("	"+operation.name+" connections count: ", operation.connections_count)
+	}
+}
+
+func print_config(ctx context.BenchCtx, tarantool_connection *tarantool.Connection, operations_map []Operaiton) {
+	fmt.Printf("%s\n", tarantool_connection.Greeting().Version)
+	fmt.Printf("Parameters:\n")
+	fmt.Printf("\tURL: %s\n", ctx.URL)
+	fmt.Printf("\tuser: %s\n", ctx.User)
+	fmt.Printf("\tconnections: %d\n", ctx.Connections)
+	fmt.Printf("\tsimultaneous requests: %d\n", ctx.SimultaneousRequests)
+	fmt.Printf("\tduration: %d seconds\n", ctx.Duration)
+	fmt.Printf("\tkey size: %d bytes\n", ctx.KeySize)
+	fmt.Printf("\tdata size: %d bytes\n", ctx.DataSize)
```

Comment:
Let's use [text/template](https://pkg.go.dev/text/template).

Reply:
We would not like to complicate the information output code yet. template requires separate squats, also with runtime error handling

Reply:
For example, you call ``PrintConfig`` in function ``Run``,  which already returns an error code. I don't see any problem in making ``PrintConfig`` function (and the like) return error, and adding just one check (in ``Run``). In my opinion, this is okay.

Reply:
I see problems with text/template https://pkg.go.dev/text/template

- we have to handle two runtime errors
- code became bigger because of additional struct
- text/template has no arithmetic, requires additional functions

Reply:
So, I think the conversation is end here. Feel free to reopen.